x-pack/plugin/ccr/src/test/java/org/elasticsearch/xpack/ccr/action/AutoFollowCoordinatorTests.java

```diff
@@ -2154,6 +2155,114 @@ void cleanFollowedRemoteIndices(ClusterState remoteClusterState, List<String> pa
         }
     }
 
+    public void testDeprecationWarningsAreEmittedWhenASystemIndexIsAutoFollowed() throws Exception {
+        // Set up a mock log appender to watch for the log message we expect
```

Comment:
This comment seems to be out of date.

Reply:
Addressed in https://github.com/elastic/elasticsearch/pull/73237/commits/b80e0e99d1f83f10afc0cbce0f85e724d92dd9b0 🤦